tutoraspects/plugin.py

```diff
@@ -214,7 +214,7 @@
         # For now we are pulling this from github, which should allow maximum
         # flexibility for forking, running branches, specific versions, etc.
         ("DBT_REPOSITORY", "https://github.com/openedx/aspects-dbt"),
-        ("DBT_BRANCH", "v1.3.1"),
+        ("DBT_BRANCH", "v2.0.0"),
```

Comment:
Will the new 2.1 break this or can it safely upgrade here?

Reply:
I just tried and it looks like it's fine. I'll update the version.